web-common/src/features/dashboards/time-series/MetricsTimeSeriesCharts.svelte

```diff
@@ -287,7 +287,10 @@
           ? $isMeasureValidPercentOfTotal(measure.name)
           : false}
 
-        <div class="flex flex-row gap-x-7">
+        <div
+          class="flex flex-row gap-x-7"
+          style:height="{expandedMeasureName ? 220 : 128}px"
+        >
```

Comment:
Is this "220" from figma mocks? TDD charts look smaller in height than before

Reply:
Will update